src/main/java/com/datadoghq/connect/logs/sink/DatadogLogsSinkTask.java

```diff
@@ -63,8 +81,15 @@ public void put(Collection<SinkRecord> records) {
                 throw new ConnectException(e);
             } else {
                 initWriter();
+                long sleepTimeMs = RetryUtil.computeRandomRetryWaitTimeInMillis(remainingRetries, config.retryBackoffMs);
```

Comment:
The first argument of `computeRandomRetryWaitTimeInMillis`  is `retryAttempts` which is not the same as `remainingRetries`.